### DISCUSSION TITLE
OPUS Codec Not Found Error on Debian Testing

### DISCUSSION THREAD

**Discussion by alishir** [baresip/baresip#3255] [Q&A]
Hi all,
I installed baresip on a Debian testing system along with the libraries related to the OPUS codec. However, when I start baresip, I get the following error: "account: audio codec not found: opus/48000/2."

I would greatly appreciate it if someone could guide me on how to resolve this issue.

Here is the list of installed packages:
```bash
dpkg -l | grep opus                                                                                                                                          
ii  libopus-dev:amd64                     1.5.2-2                              amd64        Opus codec library development files
ii  libopus0:amd64                        1.5.2-2                              amd64        Opus codec runtime library
ii  libopusenc-dev:amd64                  0.2.1-2+b2                           amd64        High-level API for encoding Opus audio streams (development files)
ii  libopusenc0:amd64                     0.2.1-2+b2                           amd64        High-level API for encoding Opus audio streams (library)
ii  libopusfile0:amd64                    0.12-4+b3                            amd64        High-level API for basic manipulation of Ogg Opus audio streams
```

**Comment by GitHubIsShortOnUsernames** [comment]
You have to define the right codec for your number of channels. I.e. opus/48000/2 for stereo and opus/48000/1 for mono. If you configure both, baresip will only use the applicable one and throw a harmless error for the other.
This is a bit confusing for Opus, because its payload is defined as always "opus/48000/2" with actual channels being requested via the fmtp line. The baresip config and payload in this case aren't the same.
